remote-state/dynamostatelock.tf

```diff
@@ -0,0 +1,15 @@
+resource "aws_dynamodb_table" "basic-dynamodb-table" {
+  name         = "statelocking-dynamodb"
+  billing_mode = "PAY_PER_REQUEST"
+  hash_key     = "LockID"
+
+  attribute {
+    name = "LockID"
+    type = "S"
+  }
+
+  tags = {
+    Name        = "dynamodb-table-1"
```

Comment:
Name tag is different from name property

Reply:
change made to name tag